### PR TITLE
chore: adds project kamp production target

### PR DESCRIPTION
**Pull request by thisislawatts**
PR Checklist

- [X] - Latest `master` branch merged
- [X] - PR title descriptive (can be used in release notes)

PR Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

Extends the CI pipeline so that it deploys to the Project Kamp production instance as well as community.preciousplastic.com each time we make a Release.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -371,7 +371,7 @@ workflows:
             - 'test_e2e'
           <<: *filter_only_master
       - deploy:
-          name: deploy_dev
+          name: "Deploy: dev.onearmy.world"
```
Comment (thisislawatts) [inline .circleci/config.yml:374]: Update name to explicitly state _where_ the deployed code will be visible. This does couple the CI pipeline to our deployed environments however as domains are updated rarely I feel that the clarity it provides makes it worthwhile.

**Comment by davehakkens** [comment]
Not sure if this is a good moment for it or worth the complexity to change.
Just throwing it here:

The original dev site is running on [dev.onearmy.world.](https://dev.onearmy.world) 
As we developed One Army we changed our main domain name to [ onearmy.earth](https://www.onearmy.earth)

So the .world domain for the dev-site is a bit off from the rest. 
Two more logical options for the dev-site would be:
1. dev.community.preciousplastic.com (if this is the dev site of PP community)
2. dev.platform.onearmy.earth (if this is the main test site for all instances)

**Comment by thisislawatts** [comment]
@davehakkens I totally agree and think that it would be worth adjusting these URLs at a later date. More immediately I think it is really helpful to decode what we mean when we say `dev_site`, because it is not obvious that dev.onearmy.world is the development version of what will become https://community.preciousplastic.com.

**Comment by thisislawatts** [comment]
But I don't think that should be a blocker to merging this PR ;) Something for future versions of us to sort out.

**Comment by davehakkens** [comment]
hehe noted!

**Review by davehakkens — APPROVED** [review]

**Pull request merged**